tianshou/highlevel/experiment.py

```diff
@@ -315,23 +355,74 @@ def run(
                     self.config.device,
                 )
 
-            # train policy
-            log.info("Starting training")
-            trainer_result: InfoStats | None = None
             if self.config.train:
                 trainer = self.agent_factory.create_trainer(world, policy_persistence)
                 world.trainer = trainer
-                trainer_result = trainer.run()
+
+        return world
+
+    def run(
+        self,
+        run_name: str | Literal["DATETIME_TAG"] | None = None,
+        logger_run_id: str | None = None,
+        raise_error_on_dirname_collision: bool = True,
+    ) -> ExperimentResult:
+        """Run the experiment and return the results.
+
+        :param run_name: if not None, will adjust the current instance's `name` name attribute.
+            The name corresponds to the directory (within the logging
+            directory) where all results associated with the experiment will be saved.
+            The name may contain path separators (i.e. `os.path.sep`, as used by `os.path.join`), in which case
+            a nested directory structure will be created.
+            If "DATETIME_TAG" is passed, use a name containing the current date and time. This option
+            is useful for preventing file-name collisions if a single experiment is executed repeatedly.
+        :param logger_run_id: Run identifier to use for logger initialization/resumption (applies when
+            using wandb, in particular).
+        :param raise_error_on_dirname_collision: set to `False` e.g., when continuing a previously executed
+            experiment with the same name.
+        :return:
+        """
+        world = self.create_experiment_world(
+            override_experiment_name=run_name,
+            logger_run_id=logger_run_id,
+            raise_error_on_dirname_collision=raise_error_on_dirname_collision,
+        )
+
+        persistence_dir = world.persist_directory
+        use_persistence = self.config.persistence_enabled
+
+        with logging.FileLoggerContext(
+            os.path.join(persistence_dir, self.LOG_FILENAME),
+            enabled=use_persistence and self.config.log_file_enabled,
+        ):
+            trainer_result: InfoStats | None = None
+            if self.config.train:
+                # prefilling buffers with random actions
```

Comment:
The comment is inaccurate; the actions are not necessarily random.
Also, break before comment recommended for readability.

Reply:
Done

Reply:
`poe format` removes the break.. :(

Maybe we can adjust the setting for this somewhere